tmt-app/src/Components/Quiz.jsx

```diff
@@ -29,6 +28,27 @@ function Quiz() {
           </Grid>
         ))}
       </Grid>
+      <Link
+        sx={{ color: 'Boba.main' }}
+        underline="none"
+        aria-label="Learn more about other different majors"
+        component={RouterLink}
+        to="/quiz/skills"
+      >
+        <PrimaryBtn
```

Comment:
Have we also played with button size variants? Rather than manual styles?

Reply:
This button was re-used from our previous components. I also believe the styling for the button size variants are different than our design.